README.md

```
@@ -12,12 +12,12 @@ This tool enables you to do such operations easily.
 
 Download the binary archive from the [releases page](https://github.com/abicky/ecsmec/releases), unpack it, and move the executable "ecsmec" to a directory in your path (e.g. `/usr/local/bin`).
 
-For example, you can install the latest binary on macOS by the following commands:
+For example, you can install the latest binary on a Mac with Apple silicon by the following commands:
 
 ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)